qemu/tests/cpu_add.py

```diff
@@ -138,8 +140,9 @@ def onoff_para_opt(onoff_params):
     vcpu_been_pluged = 0
     for i in range(vcpu_need_hotplug):
         hotplug_vcpu_params = params.object_params("hotplug_vcpu%s" % i)
-        plug_cpu_id = len(vm.vcpu_threads)
-        plug_cpu_id = hotplug_vcpu_params.get("id", plug_cpu_id)
+        plug_cpu_id = hotplug_vcpu_params.get("id")
```

Comment:
if so, you change still can fix the issue. I think better way is rename the param to avoid it conflict with preserved params. eg rename it like "vcpu_id" .

Thanks,
Xu